tests/cases/fourslash/importNameCodeFixDefaultExport3.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path="fourslash.ts" />
+
+// @allowJs: true
```

Comment:
This doesn't really need to be a JS test, right?

Reply:
While I can no longer recall my original motivation, it appears to be a JS version of the TS test in `importNameCodeFixDefaultExport2.ts`.

Reply:
Well, I don't think we need the same test twice -- the same services code will be tested either way.